equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/Crm14DetailsModel.java

```diff
@@ -10,7 +10,7 @@
 @Data
 @NoArgsConstructor
 //@EqualsAndHashCode(callSuper = true)
```

Comment:
same as above

Reply:
I will skip this because CRM14 is being worked on a different branch. But I will pass that comment to that ticket